<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>457</width>
    <height>532</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
 background-color: qlineargradient(x1: 0, y1: 1, x2: 0, y2: 0,
                                   stop: 0 #E0E0E0, stop: 1 #FFFFFF);
}
QGroupBox {
 background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                   stop: 0 #E0E0E0, stop: 1 #FFFFFF);
 border: 1px solid gray;
margin: 3px;
 border-radius: 5px;
 margin-top: 1ex; /* leave space at the top for the title */
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>441</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string>Добавление финансовой операции</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <widget class="QLineEdit" name="editDescription">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>50</y>
       <width>171</width>
       <height>32</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
     <property name="clearButtonEnabled">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>50</y>
       <width>81</width>
       <height>30</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Добавить</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>68</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Деньги</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>30</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Описание</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="editMoney">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>151</width>
       <height>30</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="suffix">
      <string> p.</string>
     </property>
     <property name="decimals">
      <number>1</number>
     </property>
     <property name="minimum">
      <double>-10000000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>10000000.000000000000000</double>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>441</width>
      <height>411</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">groupBox_2 { &quot;border: 1px solid gray;&quot; }</string>
    </property>
    <property name="title">
     <string>Операции</string>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Показать за:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboDatesAgo">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>321</width>
       <height>21</height>
      </rect>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
     <property name="currentText">
      <string/>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableTransactions">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>421</width>
       <height>341</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="gridStyle">
      <enum>Qt::SolidLine</enum>
     </property>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderShowSortIndicator" stdset="0">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderStretchLastSection">
      <bool>false</bool>
     </attribute>
    </widget>
    <widget class="QLabel" name="transactionsFor">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>10</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>10</y>
       <width>16</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>с</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>10</y>
       <width>21</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>по</string>
     </property>
    </widget>
    <widget class="QLabel" name="transactionsForTop">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>10</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
